.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,76 @@
+##############################################################################
+#
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: PR Target Workflow
+on: 
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)